src/pages/AccountRequest/accountUtil.tsx

```diff
@@ -0,0 +1,55 @@
+import { IFormOption } from "components/Form/interfaces";
+
+export interface IUserFormValues {
+  username: string;
+  email: string;
+  full_name: string;
+  role_id: number;
+  introduction: string;
+  institution_id: number;
+}
+
+export enum EmailPreference {
+  EMAIL_ON_REVIEW = "email_on_review",
+  EMAIL_ON_SUBMISSION = "email_on_submission",
+  EMAIL_ON_META_REVIEW = "email_on_review_of_review",
+}
+
+export interface IRole {
+  id: number;
+  name: string;
+  parent_id: number;
+}
+
+export interface IInstitution {
+  id: number;
+  name: string;
+}
+
+export const emailOptions: IFormOption[] = [
+  { label: "When someone else reviews my work", value: EmailPreference.EMAIL_ON_REVIEW },
+  {
+    label: "When someone else submits work I am assigned to review",
+    value: EmailPreference.EMAIL_ON_SUBMISSION,
+  },
+  {
+    label: "When someone else reviews one of my reviews (meta-reviews my work)",
+    value: EmailPreference.EMAIL_ON_META_REVIEW,
+  },
+];
+
+export const transformInstitutionsResponse = (institutionsList: string) => {
```

Comment:
As this technique is frequently employed in User pages too, may I request that you generate a new shared `utils/transformDataUtil.ts` file and consolidate this general logic in one location?